src/wandbot/evaluation/stale_eval_audit.py

```
@@ -0,0 +1,1012 @@
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
+    )
+    stdout_bytes, stderr_bytes = await process.communicate(prompt.encode("utf-8"))
+    stdout_text = stdout_bytes.decode("utf-8", errors="replace")
+    stderr_text = stderr_bytes.decode("utf-8", errors="replace")
+
+    if process.returncode != 0:
+        raise ValueError(
+            f"Codex agent failed with exit code {process.returncode}. "
+            f"stderr={stderr_text[-2000:]}"
+        )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)